src/Plugin/EntityBrowser/Widget/ParagraphEntityForm.php

```diff
@@ -45,30 +45,47 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
 
   /**
    * {@inheritdoc}
-   * 
-   * @todo fix the edit form
    */
   public function getForm(array &$original_form, FormStateInterface $form_state, array $additional_widget_parameters) {
     if (empty($this->configuration['entity_type']) || empty($this->configuration['form_mode'])) {
       return ['#markup' => $this->t('The settings for %label widget are not configured correctly.', ['%label' => $this->label()])];
     }
+    // Establish whether we're in the submit step.
+    // Either we're editing or we're in step two for a new embed.
+    $submit_step = !empty($form_state->getValues()) ? 1 : NULL;
+
+
+    // If there is an existing entity then we are editing.
+    // Set the necessary values for the paragraph update form.
+    $entity = NULL;
+    if ($entity_browser = $form_state->get('entity_browser')) {
+      /** @var \Drupal\Core\Entity\EntityInterface $entity */
+      if ($id = $entity_browser['widget_context']['current_ids']) {
```

Comment:
Undefined index: current_ids in Drupal\paragraphs_inline_entity_form\Plugin\EntityBrowser\Widget\ParagraphEntityForm->getForm() (line 63 of modules/contrib/paragraphs_inline_entity_form/src/Plugin/EntityBrowser/Widget/ParagraphEntityForm.php).